### ISSUE TITLE
[Bug]: Monal alpha crash on open

### ISSUE THREAD

**Issue by foss-** [monal-im/Monal#934]
### What Monal Release channel are you using?

Alpha

### macOS system version

13.5.1

### macOS Monal version

ID: c541d69bc03e0a4784fa57d91054432ca798022c Timestamp: 2023-08-27T15:12:46+02:00 Fix mam query bug

### Used XMPP server (domain)

not relevant

### Which XMPP-Server software are you using?

None

### How many accounts are you using in Monal?

3

### What happened?

Crash on open.

### Anything else?

Crash log (1 month): https://bin.disroot.org/?e354805bdd196f53#7BuPQvxGjyR8PG6dfVdiKrLFi4qkzjBjh5LzdS8rEY2S

### FAQ

- [X] I have [checked the FAQ](https://github.com/monal-im/Monal/wiki/FAQ---Frequently-Asked-Questions) for my issue

### Considerations for XMPP users

- [X] I have checked if my issue can be solved with [Considerations for XMPP users](https://github.com/monal-im/Monal/wiki/Considerations-for-XMPP-users) and [Considerations for XMPP server admins](https://github.com/monal-im/Monal/wiki/Considerations-for-XMPP-server-admins)

### Related Issues

- [X] I have cross-checked this overview https://github.com/monal-im/Monal/issues/322 as well as filtered for related labels https://github.com/monal-im/Monal/labels

### XEP-Check

- [X] I have checked that at least XEP-198, XEP-0280, XEP-0352, XEP-0357, XEP-0313 and XEP-0163 are activated on my server and shown as 'green' under Settings --> Account--> (i) in advanced settings

### Notifications-Menu

- [X] I have checked that all checkmarks are present under Settings --> Notifications

**Comment by tmolitor-stud-tu** [comment]
That crash is related to a muc join and mam archive query.

Did you enter a new muc?
Does this crash persist after reinstalling the app?

What was your last (muc) action before this happened?

Can you provide a logfile?

**Comment by foss-** [comment]
Did not enter a new muc. Sadly I had to remove the problematic alpha completely to prevent macOS becoming dysfunctional as disk space keep decreasing even with Monal alpha closed.

Not even sure I was actively using Monal alpha. I recall interacting with an IRC channel via disroot irc bridge `#channelname%irc.libera.chat@irc.disroot.org` and having several of such channels in chat list.

The crash was happening on every re-open of Monal alpha.

A re-install of Monal alpha resolves both the crash and the growing file issue. I suspect it was a growling log but sadly I had not verified. Was slightly hectic since not much disk space was left and space kept decreasing fairly quickly. Since I removed Monal alpha with all connected folders there should not be leftovers.

If this is not actionable feel free to close this issue.
